src/components/InfoCard.tsx

```diff
@@ -270,10 +270,33 @@ const PassedProblemsInfo = (props: {info: number[]}) => {
   );
 };
 
+const UidDisplayer = (props: {num: number}) => {
+  // definition of ".uidDisplayer" => cardStyle.ts
+  // I want CSS3 !!! /ll
```

Comment:
Well, as what I had said I will put Fluogu on hold for a while.

Reply:
I knew it, just for exercise. Now I can quickly make this stuff with CSS.
Actually, the function is from a request by @songhongyi.